CONTRIBUTING.md

```diff
@@ -1,53 +1,102 @@
+Thank you for considering contributing, whether by reporting issues,
+improving documentation, or contributing code.
+
+First, please see the [Contribution
+Guide](https://github.com/LibertyDSNP/liberty-web/blob/main/CONTRIBUTING.md)
+for Project Liberty as a whole.  Below are specifics about the Example
+Client project.
 
 ## References
-In creating bug reports, feature requests, or pull requests it's important to be familiar and up to date with both: 
+
+DSNP is a fast-moving environment.  When creating bug reports, feature
+requests, or pull requests, it's important to be familiar and
+up-to-date with:
+
 * [DSNP Specs](https://github.com/LibertyDSNP/spec) for how the DSNP works
+
 * [DSNP SDK](https://github.com/LibertyDSNP/sdk-ts) for integration with the DSNP
 
 ## Reporting
 
 ### Bugs
-If you are using the example client and you find a bug that you believe should be fixed, please create a new issue on our [Github Issues Page](https://github.com/LibertyDSNP/example-client/issues). Make sure to **tag it as a bug**, and follow the `Bug Report Template`. For all intents and purposes a Bug is any one of the following things:
-* Broken/Nonfunctional code
-* Unexpected/Unintentional results
-* Code vulnerabilities
-* Out of date code/packages/documentation
-* Documentation/Code inconsistencies
+
+If you are using the Example Client and you find a bug that you
+believe should be fixed, please [create a new issue
+ticket](https://github.com/LibertyDSNP/example-client/issues/new/choose)
+(after first [looking to see if there is an existing
+issue](https://github.com/LibertyDSNP/example-client/issues) about it,
+of course).  Make sure to **tag the new issue as a bug**, and follow
+the `Bug Report` template.
+
+A "bug" is any one of the following things:
+
+* Broken or non-functional code
+* Unexpected or clearly unintentional results
+* Security vulnerabilities
+* Out-of-date code/dependencies/documentation
+* Inconsistencies in the documentation or the code
 
 ### Features
-If you are using the example client and would like to request a non-bug related change or a new feature not yet included, please create a new issue on our [Github Issues Page](https://github.com/LibertyDSNP/example-client/issues). Make sure to **tag it as a feature request**, and follow the `Feature Request Template`. For all intents and purposes a Feature Request is any one of the following things:
-* New example client functionality 
+
+To suggest an enhancement or new feature, please [create a new issue
+ticket](https://github.com/LibertyDSNP/example-client/issues/new/choose)
+(after first [looking to see if anyone else has suggested the same
+thing](https://github.com/LibertyDSNP/example-client/issues)).  Make
+sure to **tag it as a feature request**, and follow the `Feature
+Request` template.
+
+A "feature request" is any one of the following things:
```

Comment:
Similarly I don't think we need to define what a feature request is.

Reply:
Likewise good point :-).